Sources/Commands/SwiftTool.swift

```diff
@@ -301,11 +301,6 @@ public class SwiftTool<Options: ToolOptions> {
                 usage: "Increase verbosity of informational output"),
             to: { $0.verbosity = $1 ? 1 : 0 })
 
-        binder.bind(
-            option: parser.add(option: "--no-static-swift-stdlib", kind: Bool.self,
-                usage: "Do not link Swift stdlib statically"),
```

Comment:
Instead of removing this, I think we should change the help to say that this is the default option. IIRC this was introduced so people have option to opt of static stdlib if we make it the default in future.

```
Do not link Swift stdlib statically [default]
```

Reply:
That makes sense, I've added it back and updated the text